src/mlmc/sampling_pool_pbs.py

```diff
@@ -16,21 +16,20 @@ class SamplingPoolPBS(SamplingPool):
     JOBS_DIR = "jobs"
     LEVEL_SIM_CONFIG = "level_{}_simulation_config"  # Serialized level simulation
     JOB = "{}_job.sh"  # Pbs process file
+    JOBS_COUNT = "jobs_count.txt" # Contains current number of jobs which is also job unique identifier
```

Comment:
If we need this, can we store with some other information, not introducing a new file for that?
But possibly we can get this only by listing the 'jobs' directory and getting all "*_job.sh" job scripts.